assets/js/components/user-input/UserInputPreview.js

```diff
@@ -86,13 +77,12 @@ export default function UserInputPreview( props ) {
 		USER_INPUT_ANSWERS_POST_FREQUENCY,
 		USER_INPUT_ANSWERS_GOALS,
 	} = getUserInputAnswers();
+
 	const [ page ] = useQueryArg( 'page' );
 
-	const [ errorMessages, setErrorMessages ] = useState( {
-		[ USER_INPUT_QUESTIONS_GOALS ]: null,
-		[ USER_INPUT_QUESTIONS_PURPOSE ]: null,
-		[ USER_INPUT_QUESTION_POST_FREQUENCY ]: null,
-	} );
+	const hasError = USER_INPUT_QUESTIONS_LIST.some( ( slug ) =>
```

Comment:
If there is error in any question, the save button should be disabled, not silently fail.

Reply:
Makes sense 👍